ingest-app/docs/api.md

```diff
@@ -679,7 +679,11 @@ To disable validation of the parsed UMM metadata against the UMM spec, pass `ski
 Example: Translate ECHO10 metadata to UMM JSON
 
 ```
-curl -i -XPOST -H "Content-Type: application/echo10+xml" -H "Accept:  application/vnd.nasa.cmr.umm+json;version=1.2" %CMR-ENDPOINT%/translate/collection?skip_umm_validation=true -d \
+curl -i -XPOST \
+  -H "Content-Type: application/echo10+xml" \
+  -H "Accept: application/vnd.nasa.cmr.umm+json;version=1.14" \
```

Comment:
I question why we even specify a version number here, I can pull it out.

Reply:
This demonstrates that user can specify a specific UMM version as the target output format. We should make that clearer in the api doc.